### PR TITLE
Enhanced Responsiveness of Cytology and Pathology Dashboard

### PR DESCRIPTION
**Pull request by PR4NJ41**
### Description:
Previously, the pathology and cytology dashboard pages displayed four tiles in a row within the grid, leading to overflow issues. This pull request addressed the problem by adjusting the layout so that for mobile views, only two tiles are shown per row, thereby preventing overflow and improving the page's responsiveness.

### Before:
Mobile View
Screenshot from 2024-03-20 05-19-48
Screenshot from 2024-03-20 05-32-13

Desktop View
Screenshot from 2024-03-20 05-18-06
Screenshot from 2024-03-20 05-18-18

### After:
Mobile View
Screenshot from 2024-03-20 05-17-58
Screenshot from 2024-03-20 05-18-26

Desktop View
Screenshot from 2024-03-20 05-18-06
Screenshot from 2024-03-20 05-18-18

### CODE REVIEW

**Comment by PR4NJ41** [comment]
@mozzy11 Sir, I have made the required changes to my PR. Please review it. Thank You

**Comment by PR4NJ41** [comment]
@mozzy11 Sir, Please review my PR. Thank You

**Comment by mozzy11** [comment]
Thanks @PR4NJ41 .
I dont seem to see any breadcrupbs on your screenshot ??

**Comment by mozzy11** [comment]
can you resolve the merge conflicts /?

**Comment by PR4NJ41** [comment]
> Thanks @PR4NJ41 . I dont seem to see any breadcrupbs on your screenshot ??
Sorry, there was some problem in my system while reloading the page. The changes that I have made , have nothing to do with breadcrumbs, so they are unaffected. I have edited my PR comment with the right images.

File: frontend/src/components/pathology/PathologyDashboard.css [hunk 1]
```diff
@@ -71,37 +71,39 @@
...
-    min-width: 175px;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 20px;
+    padding:20px;
   }
-  .tile-title-Immuno{
+  .tile-title {
     font-size: 0.9rem;
-  }
 }
```
[the remark this answers is not in the document]
Comment (PR4NJ41) [reply frontend/src/components/pathology/PathologyDashboard.css]: Hey @atulyadav745, I appreciate your review of my pull request. I have already tried what you are suggesting, I was still encountering overflow issues on mobile view. Certain tiles are displaying long words that extend beyond the boundaries of the tile, so I have adjusted font size and other parameters accordingly.

**Comment by PR4NJ41** [comment]
> can you resolve the merge conflicts /?

@mozzy11 Sir, I have resolved the merge conflicts. Please, review my PR.

**Review by mozzy11 — APPROVED** [review]

**Pull request merged**